### DISCUSSION TITLE
Error help ABM student

### DISCUSSION THREAD

**Discussion by AmberAgyemang** [mesa/mesa#2577] [Q&A]
HI community!

I am using the mesa package for my project for the ABM course at TU Delft and I have run into an error and I was wondering if there is anyone out there that could help me.

![Screenshot 2024-12-29 141343](https://github.com/user-attachments/assets/9fbdbf63-1038-4547-85a8-4df9f8ae98f6)

The error occurs in a line of the model that was written by the teachers of the course.

Thank you in advance!

**Comment by quaquel** [comment]
There is not enough information here at all to figure out what is going on. It is clear that some variable called pos is a single number rather than a sequence of length 2. However, without any further information, it is impossible to tell why this is the case.

**Comment by AmberAgyemang** [comment]
Solved, but thanks for the effort!
